code/modules/mob/living/carbon/carbon.dm

```diff
@@ -200,7 +200,7 @@
 					num_injuries++
 
 			if(num_injuries == 0)
-				to_chat(src, "My legs are OK.")
+				to_chat(src, "My limbs are okay.")
```

Comment:
>okay

Reply:
![no_difference_between_ok_and_okay](https://user-images.githubusercontent.com/30557196/47260584-241fcb80-d4b6-11e8-95d5-945f07cf10b4.PNG)

Reply:
REMOVE ISLANDER remove islander you are worst islander

Reply:
I was joking and don't actually give a shit but [the spelling "OK" predates the spelling "okay" by centuries](https://books.google.com/ngrams/graph?content=ok%2Cokay&case_insensitive=on&year_start=0&year_end=9999&corpus=15&smoothing=3&share=&direct_url=t4%3B%2Cok%3B%2Cc0%3B%2Cs0%3B%3BOK%3B%2Cc0%3B%3Bok%3B%2Cc0%3B%3BOk%3B%2Cc0%3B%3BoK%3B%2Cc0%3B.t4%3B%2Cokay%3B%2Cc0%3B%2Cs0%3B%3Bokay%3B%2Cc0%3B%3BOkay%3B%2Cc0)